### PR TITLE
Fix flow

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers @chrisnojima 

Now we have two flow things. A flow that lives under `desktop` and another that lives under `react-native`.

The reason for the difference is so that things in `desktop/shared` can reference `../desktop`. 

In `react-native` flow we ignore .desktop files.

it might be possible to remap the path `../desktop` so that it resolves when under react-native, thoughts on whether this should be done?

### CODE REVIEW

File: desktop/.flowconfig [hunk 1]
```diff
@@ -0,0 +1,86 @@
+[ignore]
+
+# TODO: unignore react-native when we upgrade to .16
```
Comment (chrisnojima) [inline desktop/.flowconfig]: do we need this?

File: desktop/.flowconfig [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+.*/shared/packager/.*
+.*/shared/purepack/.*
+.*/shared/framed-msgpack-rpc/.*
+.*/shared/iced-error/.*
+.*/shared/debug/.*
+
+# Ignore .*-render.platform.js files
+.*-render\.android\.js
+.*-render\.ios\.js
+.*-render\.mobile\.js
```
Comment (chrisnojima) [inline desktop/.flowconfig]: this is .native now right?

**Comment by chrisnojima** [comment]
:+1:

**Comment by MarcoPolo** [comment]
This didn't quite work, new PR incoming!

**Pull request closed without merging**